backend/seed_data/quiz_questions.yaml

```
@@ -1212,3 +1212,495 @@ quiz_questions:
+    title: "What is the degree of a vertex in a graph?"
+    options:
+      - option_text: "The number of edges connected to the vertex"
+        is_correct: true
+        option_type: CONCEPT
+      - option_text: "The maximum distance from this vertex to any other vertex"
+        is_correct: false
+        option_type: CONCEPT
+      - option_text: "The number of paths from the vertex to other vertices"
+        is_correct: false
+        option_type: CONCEPT
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)